### ISSUE TITLE
Autofocus: Screen reader not reading list items in auto complete

### ISSUE THREAD

**Issue by rinnuagna** [primefaces/primereact#7399]
### Describe the bug

When screen reader moves through the list items of autocomplete, it will read as blank if no value in autocomplete, or repeatedly read the selected value for all the list items 
![image](https://github.com/user-attachments/assets/c6cd57ef-d053-4cef-92ca-416ec139faac)

### System Information

```Shell
primereact 10.8.3
```

### Expected behavior

All the items in the list should be read by screen reader same how dropdown works

**Comment by danielck** [comment]
I tested this component in a few screenreaders (latest NVDA and JAWS 2024 both running Chrome, and VoiceOver on macOS 15.1 with Safari) and none of them work. I can confirm the behaviour reported by @rinnuagna in NVDA.

The accessibility of comboboxes is very flaky even when implemented according to spec, but a possible reason for the issues is that aria-activedescendant is not set or updated on the input (which has the combobox role). The documentation claims the attribute is there, so this must be a bug.

The listbox does not follow the input in the DOM, which can be another cause of trouble but likely not the main issue here.

I'm happy to retest any fixes.
